baseapp_follows/migrations/0008_followstats_alter_follow_actor_alter_follow_target.py

```
@@ -0,0 +1,62 @@
+        ),
+        migrations.AlterField(
+            model_name="follow",
+            name="target",
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="followers",
+                to="baseapp_core.documentid",
+                verbose_name="target",
+            ),
+        ),
+    ]
```

Ruff (0.15.6)

[warning] Mutable default value for class attribute

(RUF012)